src/main/java/sky/pro/course2/coursework/repository/JavaQuestionRepository.java

```diff
@@ -0,0 +1,59 @@
+package sky.pro.course2.coursework.repository;
+
+import org.springframework.stereotype.Repository;
+import sky.pro.course2.coursework.model.Question;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+@Repository
+public class JavaQuestionRepository implements QuestionRepository {
+    private final Set<Question> questions = new HashSet<>();
+
+    @Override
+    public Question add(String question, String answer) {
+        if (checkInput(question, answer)) {
+            Question newQuestion = new Question(question, answer);
+            this.questions.add(newQuestion);
+            return newQuestion;
+        } else {
+            throw new IllegalArgumentException("Требуется ввести корректные данные (вопрос - ответ)");
+        }
```

Comment:
```suggestion
        return add(new Question(question, answer));
```
надо просто вызвать уже написанный метод, а не дублировать код